app/src/main/res/layout/fragment_impact.xml

```diff
@@ -1,26 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:tools="http://schemas.android.com/tools"
+<LinearLayout
     android:layout_height="match_parent"
     android:layout_width="match_parent"
     xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto">
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical">
 
     <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/rvChallenges"
         android:layout_width="match_parent"
-        android:layout_height="@dimen/impact_fragment_recycler_view_height"
+        android:layout_height="0dp"
         app:layout_constraintEnd_toEndOf="parent"
```

Comment:
you can get rid of the constraints as you are no longer using `androidx.constraintlayout.widget.ConstraintLayout`